projects/js-packages/publicize-components/src/components/admin-page/toggles/styles.module.scss

```diff
@@ -26,6 +26,10 @@
 	}
 }
 
+.column.notoggle {
+	display:block;
+}
+
```

Comment:
This seems like a bit of a hack to me. I've added it as I want this section to take up the grid column where the toggle would be if there is no toggle. 

The problem is that I think the width of the `div` is calculated from all the grid layout directives, so this becomes a bit of weird hybrid. There's probably a better way to do it by adjusting the grid properties.

Reply:
Ok, I think I've fixed this now.